lxd/instance/drivers/driver_qemu.go

```diff
@@ -1221,6 +1221,13 @@ func (d *qemu) start(stateful bool, op *operationlock.InstanceOperation) error {
 	// Define a set of files to open and pass their file descriptors to qemu command.
 	fdFiles := make([]*os.File, 0)
 
+	// Ensure passed files are closed after start has returned (either because qemu has started or on error).
```

Comment:
This change and the removal of the defer below should be a separate commit as its not really related to SEV change and is a good candidate for back porting to LTS branch (where new features are not backported), so we don't want to tightly couple it to the SEV change.